kansha/services/assetsmanager.py

```diff
@@ -69,3 +79,31 @@ def data_for_src(self, file_id):
         """
         data, metadata = self.load(file_id, True)
         return "data:image/gif;base64,%s" % data.encode('base64')
+
+
+@presentation.init_for(AssetsManager, "len(url) >= 2")
+def init_assets(self, url, comp, http_method, request):
+    headers = {}
+    metadata = self.get_metadata(url[0])
+    try:
+        headers['content_type'] = metadata['content-type']
+    except KeyError:
+        pass
+
+    raise FileResponse(self._get_filename(url[0],
```

Comment:
Are we sure 'FileResponse()' will return '404' if the filename doesn't exist? Else, we must check the existence here.